src/search-bar/search.css

```diff
@@ -0,0 +1,64 @@
+
+@import 
+url('https://fonts.googleapis.com/css2?family=Rubik:ital,wght@0,300..900;1,300..900&display=swap');
+
+.logo{
+    float: left;
+}
+
+.logo:active, .enteredtext{
+    animation: shake 0.4s linear 0s ;
+}
+
+
+ .logo:hover{
+    width: 18px;
+    height: 18px;
+
+}
+
+.text{
```

Comment:
not mandatory but using a very general class name makes the code harder to read.